backend/globaleaks/handlers/receiver.py

```diff
@@ -86,7 +86,7 @@ def get_receivertip_list(store, receiver_id, language):
             'preview_schema': db_get_archived_preview_schema(store, rtip.internaltip.questionnaire_hash, language),
             'preview': rtip.internaltip.preview,
             'total_score': rtip.internaltip.total_score,
-            'label': rtip.label
```

Comment:
i think this is a wrong merge of a comma i removed; going to remove it while mergin the pull request.